lib/publiccloud/aws.pm

```diff
@@ -16,6 +16,7 @@ use Mojo::Base 'publiccloud::provider';
 use Mojo::JSON 'decode_json';
 use testapi;
 use publiccloud::utils "is_byos";
+use utils "zypper_call";
```

Comment:
Do we use `zypper_call` in this library?

Reply:
Looks like some left over. Can you remove it?

Reply:
And `use publiccloud::utils "is_byos";` is not used neither

Reply:
done